lib/airbrake.js

```diff
@@ -138,61 +138,61 @@ Airbrake.prototype._sendRequest = function(err, cb) {
     headers: {
       'Content-Length': body.length,
       'Content-Type': 'application/json',
-      'Accept': 'application/json',
-    },
+      Accept: 'application/json'
+    }
   }, this.requestOptions);
 
-  request(options, function(err, res, body) {
-    if (err) {
+  request(options, function (requestErr, res, responseBody) {
+    if (requestErr) {
       return callback(err);
     }
 
-    if (undefined === body) {
+    if (typeof responseBody === 'undefined') {
```

Comment:
This looks like a style win as well as being safer. Can you explain the difference between this and say

```
if (typeof responseBody ===  typeof undefined)
```

Which is better?  

Or did you use the `'undefined'` string because Lint fails with `typeof undefined`?